tools/hive-bigquery/utilities.py

```diff
@@ -0,0 +1,89 @@
+"""Provides utility functions to other modules"""
+
+import logging
+import random
+import string
+import subprocess
+
+import tableprint as tp
+
+logger = logging.getLogger('Hive2BigQuery')
+
+
+def calculate_time(start, end):
+    """Pretty prints the time taken for an operation
+
+    Args:
+        start (float): Start time of an operation
+        end (float): End time of an operation
+
+    Returns:
+        str: Pretty format the time taken for an operation
+    """
+
+    time_taken = int(round((end - start), 0))
+    day = time_taken // 86400
+    hour = (time_taken - (day * 86400)) // 3600
+    minutes = (time_taken - ((day * 86400) + (hour * 3600))) // 60
+    seconds = time_taken - ((day * 86400) + (hour * 3600) + (minutes * 60))
+
+    if day != 0:
+        output = '{} days {} hours {} min {} sec'.format(day, hour, minutes,
+                                                         seconds)
+    elif hour != 0:
+        output = '{} hours {} min {} sec'.format(hour, minutes, seconds)
+    elif minutes != 0:
+        output = '{} min {} sec'.format(minutes, seconds)
+    else:
+        output = '{} sec'.format(seconds)
+
+    return output
+
+
+def print_and_log(output, level=logging.DEBUG):
+    """Pretty prints the statement and logs it to the log file using the
+    provided log level
+
+    Args:
+        output (str): statement to be logged
+        level (int): log level
+    """
+
+    tp.banner(output)
+    if level == logging.INFO:
+        logger.info(output)
+    elif level == logging.ERROR:
+        logger.error(output)
+    elif level == logging.WARNING:
+        logger.warning(output)
+    elif level == logging.CRITICAL:
+        logger.critical(output)
+    else:
+        logger.debug(output)
+
+
+def get_random_string():
```

Comment:
remove and use `uuid.uuid4()` it is safer and clearer what you're trying to achieve.